### ISSUE TITLE
class collapse inside inline not working any more

### ISSUE THREAD

**Issue by omargawdat** [unfoldadmin/django-unfold#816]
unfold: 0.40 
django: 5.12 
chrome: latest 

description: the collapse inside inline isn't working anymore

code:

```
class WorkerInline(StackedInline):
    form = WorkerForm
    model = Worker
    fk_name = "company"
    fieldsets = (
        (
            "Personal Information",
            {
                "fields": (
                    ("first_name", "last_name"),
                    "phone_number",
                    "gender",
                    "image",
                    "packages",
                    "is_active",
                )
            },
        ),
    )
    filter_horizontal = ("packages",)
    classes = ["collapse"]

```

additionally when I tried to use tab = true it doesn't work when switching language 
english: 
<img width="1136" alt="image" src="https://github.com/user-attachments/assets/59f72a77-b016-4442-97d5-1b24a81c1f1b">
arabic: 
<img width="1097" alt="image" src="https://github.com/user-attachments/assets/15acd734-843c-4bb1-96eb-30753651d95b">

**Comment by radzihcake** [comment]
+

**Comment by lukasvinclav** [comment]
Fixed in #870
